package.json

```diff
@@ -84,6 +84,7 @@
     "extract-text-webpack-plugin": "^1.0.1",
     "file-loader": "^0.9.0",
     "helmet": "^2.1.2",
+    "html-loader": "^0.4.4",
```

Comment:
Given that we're no longer interpolating `require`, and pushing all the configurations into `headAssets`.

It seems like we will no longer need `html-loader`. 

We could instead write a function 

```javascript
const createTemplate = ({ meta, link, title, script }) => (
`
<html>
   <head>
      ${title}
      ${meta}
      ${link}
   </head>
</html>
`
)
```

Unless you can see a benefit in keeping html-loader?